lib/react.dart

```diff
@@ -139,6 +172,8 @@ abstract class Component {
   ///
   /// [A.k.a "forceUpdate"](https://facebook.github.io/react/docs/react-component.html#forceupdate)
   void redraw([callback()]) {
+    context = getChildContext();
```

Comment:
Why does this happen here but not in `setState`?

Reply:
Hrmm...guess I mistook JS for Dart here w/ the `_jsRedraw` vs `redraw`.

Should it go in `replaceState` as well?

Reply:
Hmm, what's the purpose of doing this anyways? Doesn't this https://github.com/cleandart/react-dart/pull/135/files#diff-12ac054bb8b76f35c1035afef1d94396R257 update the context?

Reply:
Great find. I removed it and all tests pass + examples work. See 723caa5.